arm-resources/locks/2015-01-01/swagger/locks.json

```diff
@@ -547,9 +588,6 @@
             "type": "string",
```

Comment:
Why have we modeled this operation in the first place? This will automatically be provided if the actual list operation has the "x-ms-pageable" extension applied to it.
```
 "/{nextLink}": {
       "get": {
         "tags": [
           "ManagementLocks"
         ],
         "operationId": "ManagementLocks_ListNext",
```